apisix/plugins/openid-connect.lua

```diff
@@ -293,30 +293,28 @@ function _M.rewrite(plugin_conf, ctx)
         -- the configured redirect URI after successful authentication.
         response, err = openidc.authenticate(conf)
 
-        if err then
+        if not response then
```

Comment:
the openidc returns non-nil err to indicate something wrong, see:
https://github.com/zmartzone/lua-resty-openidc#sample-configuration-for-google-signin
https://github.com/zmartzone/lua-resty-openidc/blob/master/lib/resty/openidc.lua#L1494

Reply:
Thanks. Updated